manifests/k8s/config/exporter/patch/patch-machine-spec.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kepler-exporter
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - name: kepler-exporter
+          env:
+            - name: MACHINE_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['kepler-profile']
```

Comment:
Would't this set the same profile for all nodes that kepler is deployed to? 
Consider a cluster with different types of nodes. Since label and value on the daemonset is going to be the same, this would force all pods (deployed on each node) to have the same machine spec loaded. 

I think we need to rethink the idea of using daemonsets for kepler since we will want to load node-specific configuration for each pod.

Reply:
For the current design, we need one kepler ds for each group of node (with node selector).

Reply:
Isn't that equal to using `Pod` instead of `Daemonset`?

Reply:
Is this relevant anymore? shouldn't`--machine-profile` be added to the args instead?